pkg/cli/baseimage.go

```diff
@@ -44,15 +44,34 @@ func NewBaseImageRootCommand() (*cobra.Command, error) {
 		newBaseImageDockerfileCommand(),
 		newBaseImageBuildCommand(),
 		newBaseImageGenerateMatrix(),
+		newVersionsMatrix(),
 	)
 
 	return &rootCmd, nil
 }
 
+func newVersionsMatrix() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "generate-versions",
+		Short: "Generate a nested matrix of python, cuda, pytorch versions (json)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			versions := dockerfile.BaseImageConfigurations()
+			output, err := json.Marshal(versions)
+			if err != nil {
+				return err
+			}
+			fmt.Println(string(output))
+			return nil
+		},
+		Args: cobra.MaximumNArgs(0),
+	}
+	return cmd
+}
+
```

Comment:
What's the difference between this new command and the existing `generate-matrix` command implemented below?

Reply:
As far as I can tell, `generate-versions` was an unused duplicate, so I removed it with 6d6d54c4ed0e018418d382795a8ba24947cbac24